Makefile

```diff
@@ -78,7 +78,6 @@ python:
 	$(MAKE) -C scripts/examples/python build
 
 doc-json:
-	dune build --profile=$(PROFILE) ocaml/idl/json_backend/gen_json.exe
```

Comment:
Will `dune exec` below always rebuild this?

Reply:
It will rebuild as needed, as it was previously